YehorTask/Phonebook.cs

```diff
@@ -0,0 +1,203 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace YehorTask
+{
+  class Phonebook
+  {
+    #region Поля и свойства
+    /// <summary>
+    /// Словарь который связывает имя с ISubscriber.
+    /// </summary>
+    private Dictionary<string, List<ISubscriber>> mapByName;

```

Comment:
```suggestion
    private List<ISubscriber> subsribers;

```
предлагаю переделать на список абонентов

Reply:
Исправил.